File: Javascript/Konverter Mata Uang.js
// Kurs tetap (contoh: 1 USD = 15000 IDR, 1 EUR = 16000 IDR)
const rates = {
    USD: { IDR: 15000, EUR: 0.85 },
    EUR: { IDR: 16000, USD: 1.18 },
    IDR: { USD: 1/15000, EUR: 1/16000 }
};

function convertCurrency(amount, from, to) {
    if (!rates[from] || !rates[from][to]) {
        return "Konversi tidak didukung.";
    }
    return amount * rates[from][to];
}

// Fungsi utama
function currencyConverter() {
    console.log("Selamat datang di Konverter Mata Uang!");
    console.log("Mata uang yang didukung: USD, EUR, IDR");

    const amount = parseFloat(prompt("Masukkan jumlah:"));
    const from = prompt("Dari mata uang (USD/EUR/IDR):").toUpperCase();
    const to = prompt("Ke mata uang (USD/EUR/IDR):").toUpperCase();

    if (isNaN(amount)) {
        console.log("Masukkan jumlah yang valid!");
        return;
    }

    const result = convertCurrency(amount, from, to);
    console.log(`${amount} ${from} = ${result} ${to}`);
}

// Jalankan konverter
currencyConverter();

File: Python/Konverter Mata Uang.py
def konverter_mata_uang():
    print("\n" + "$" * 25)
    print("  KONVERTER RUPIAH (IDR)  ")
    print("$" * 25)
    
    try:
        idr = float(input("Masukkan jumlah Rupiah (Rp): "))
    except ValueError:
        print("❌ Masukkan angka saja ya!")
        return

    print("\nPilih target mata uang:")
    print("1. USD (Dollar AS) - Kurs: 15.500")
    print("2. JPY (Yen Jepang) - Kurs: 105")
    print("3. SGD (Dollar Singapura) - Kurs: 11.500")
    
    pilihan = input("Pilih (1-3): ")
    
    if pilihan == '1':
        hasil = idr / 15500
        print(f"\n💵 Hasil: $ {hasil:.2f} USD")
    elif pilihan == '2':
        hasil = idr / 105
        print(f"\n💴 Hasil: ¥ {hasil:.2f} JPY")
    elif pilihan == '3':
        hasil = idr / 11500
        print(f"\n🇸🇬 Hasil: $ {hasil:.2f} SGD")
    else:
        print("❌ Pilihan tidak tersedia.")

# Jalankan konverter
konverter_mata_uang()